commit 24c2adf0cbd4
Author: Michael Kaply <mozilla@kaply.com>
Date:   Fri Jun 1 16:59:03 2018 -0500

    Bug 1455261 - Check for null metadata to workaround broken search.json. r=adw, a=RyanVM
    
    MozReview-Commit-ID: 4D2F30MHaqE
    
    --HG--
    extra : source : 9a5f5fa5cfa0a6fe45eb7685fb1798529e8f406a
---
 toolkit/components/search/nsSearchService.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git toolkit/components/search/nsSearchService.js toolkit/components/search/nsSearchService.js
index 6c153c5e29cdf..86a286819f02b 100644
--- toolkit/components/search/nsSearchService.js
+++ toolkit/components/search/nsSearchService.js
@@ -2074,7 +2074,7 @@ Engine.prototype = {
   },
 
   getAttr(name) {
-    return this._metaData[name] || undefined;
+    return (this._metaData && this._metaData[name]) || undefined;
   },
 
   // nsISearchEngine
